### DISCUSSION TITLE
How to parse JSON in new Postman Flow and to send 1 field to Terminal

### DISCUSSION THREAD

**Discussion by vitaliCoasy** [postmanlabs/postman-flows#209] [Q&A]
Previously it was "Parse Json" step in postman flow. Now, I dont see it. I need to parse the Response's JSON and to send 1 field to Terminal to show (running hundreds of requests at the same time). How it can be done now?

**Comment by Silenco** [comment]
Same issue here!

**Comment by samay-v** [comment]
Hey, you can use the "create data" block to create either a record/list or fields you want to send:
![image](https://user-images.githubusercontent.com/28357106/154127516-c5c19c5d-88c9-4fc3-a5b3-09b8cc80ebac.png)

Or add only one field 
![image](https://user-images.githubusercontent.com/28357106/154127618-cd6b6975-5289-49c7-8957-261e623d7db8.png)

You can then connect this to the terminal to achieve your usecase.

**Reply by alexei-moneyr** [reply]
its not letting me to make reference to /data/body/status. only as a string

**Reply by sotolou2022** [reply]
Yea this does not make any sense at all, frustrating..

**Reply by denik1981** [reply]
Same here. I believe this feature is not workings still.

**Reply by mitchell-johnson** [reply]
I to am having issues with this feature. In my case I want to call a few api's and dump their output ideally I would just like to pump it to a json file but the terminal would do but I seem to just get everything output to the terminal.

**Reply by mattnield** [reply]
Is the output in the Console ok for what you need @mitchell-johnson 

Overall, more control of _what_ goes to the terminal without having to create variables would be great!
